includes/handlers/class-webmention-handler-jsonld.php

```diff
@@ -0,0 +1,111 @@
+<?php
+/**
+ * Class for webmention parsing using JSON-LD.
+ */
+class Webmention_Handler_JSONLD extends Webmention_Handler_Base {
+
+	/**
+	 * Handler Slug to Uniquely Identify it.
+	 *
+	 * @var string
+	 */
+	protected $slug = 'jsonld';
+
+	/**
+	 * Takes a request object and parses it.
+	 *
+	 * @param Webmention_Request $request Request Object.
+	 * @param Webmention_Item $item A Parsed Item. If null, a new one will be created.
+	 * @return WP_Error|true Return error or true if successful.
+	 */
+	public function parse( $request, $item = null ) {
+		if ( $item instanceof Webmention_Item ) {
```

Comment:
this is used by every parse function, I would like to refactor this. Should we really keep this as param?